cairo/ethereum/cancun/vm/instructions/block.cairo

```diff
@@ -302,6 +302,7 @@ namespace Internals {
                 return err;
             }
         }
+        EvmImpl.set_stack{evm=evm}(stack);
```

Comment:
this is useless because evm is not returned. Actually you should remove all                 `EvmImpl.set_stack{evm=evm}(stack);` from this function as stack is not taken from the evm.